docs/contributing/orbstack.md

```diff
@@ -0,0 +1,39 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+-->
+
+# OrbStack as a docker alternative on macOS
+1. Install OrbStack by downloading [installer](https://orbstack.dev/download) or using Homebrew.
+    ```shell
+    brew install orbstack
+    ```
+   
+2. Migrate Docker data
+    ```shell
+    orbstack migrate docker
+    ```
+   
+3. (Optional) Add registry mirrors
+   
+    You can edit the config directly at `~/.orbstack/config/docker.json` and restart the engine with `orb restart docker`.
+   
+    ```
+    {
+        "registry-mirrors": ["https://registry.docker.ir", "https://docker.iranserver.com"]
```

Comment:
nit: why are these registry mirrors part of the PR? The only reference i can find about these registry mirrors is here https://github.com/orbstack/orbstack/issues/51#issuecomment-2390824878  

It doesnt seem like these mirrors are commonly used
* https://grep.app/search?q=docker.iranserver.com
* https://grep.app/search?q=registry.docker.ir

can we remove this recommendation or change it to a more commonly used mirror? 
cc @liurenjie1024 @lewiszlw

Reply:
These registry mirrors are just example, copied from https://github.com/orbstack/orbstack/issues/51#issuecomment-2390824878 . I think we can change it like 
```
{
        "registry-mirrors": ["<mirror_addr>"]
}
```

Reply:
that sounds good to me, thanks! 

Let's see if I can change it right here from github:
```suggestion
        "registry-mirrors": ["<mirror_addr>"]
```